charts/hono/Chart.yaml

```diff
@@ -15,7 +15,7 @@ name: hono
 description: |
   Eclipse Hono™ provides remote service interfaces for connecting large numbers of IoT devices to a back end and
   interacting with them in a uniform way regardless of the device communication protocol.
-version: 1.10.19
+version: 1.10.20
```

Comment:
The chart is already at 1.10.20 so you will need to bump it to 1.10.21 ...

Reply:
Ok, I've missed it. It's rebased and fixed with version 1.10.21 now.